fpr/migrations/0019_mediaconch_validation_verbosity.py

```diff
@@ -0,0 +1,86 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
+import os
+
+from django.db import migrations, models
+
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+MIGR_DATA = os.path.join(os.path.dirname(HERE), 'migrations-data')
+VALIDATE_CMD_FNM = os.path.join(MIGR_DATA, 'mc_validate_cmd_2.py')
+
+
+def data_migration(apps, schema_editor):
+    """Modify the MediaConch-related validation objects in order to produce
+    less verbose output. This involves:
+
+    - disabling the tool, command and rule introduced by migration 0011,
+    - creating a new FPTool for MediaConch v. 17.12 (which does not hang when
+      validating certain .mkv files),
+    - creating a new validation FPCommand that uses the 17.12 tool, and
+    - creating a new mkv validation FPRule that uses the new command.
+    """
+
+    FPTool = apps.get_model('fpr', 'FPTool')
+    FPCommand = apps.get_model('fpr', 'FPCommand')
+    FPRule = apps.get_model('fpr', 'FPRule')
+    FormatVersion = apps.get_model('fpr', 'FormatVersion')
+    mkv_format = FormatVersion.objects.get(description='Generic MKV')
+
+    # Disable the MediaConch v. 16.12 FPTool
+    old_mediaconch_tool_uuid = 'f79c56f1-2d42-440a-8a1f-f40888e24bca'
+    FPTool.objects.filter(uuid=old_mediaconch_tool_uuid).update(enabled=False)
+
+    # Create a new MediaConch v. 17.12 FPTool
+    new_mediaconch_tool_uuid = '9e78669d-e2e5-490d-a783-54f4325ab950'
+    new_mediaconch_tool = FPTool.objects.create(
+        uuid=new_mediaconch_tool_uuid,
+        description='MediaConch',
+        version='17.12',
+        enabled=True,
+        slug='mediaconch-1712'
+    )
+
+    # Disable the old MediaConch Validation FPR Command
+    old_mediaconch_command_uuid = '287656fb-e58f-4967-bf72-0bae3bbb5ca8'
+    FPCommand.objects.filter(uuid=old_mediaconch_command_uuid).update(
+        enabled=False)
+
+    # Create a new MediaConch Validation FPR Command that contains the new
+    # validation script which reduces the verbosity level.
+    with open(VALIDATE_CMD_FNM) as filei:
+        mediaconch_command_script = filei.read()
+    new_mediaconch_command_uuid = 'eab36a97-7b15-42a0-a6dc-29e1de62ac61'
+    new_mediaconch_command = FPCommand.objects.create(
+        uuid=new_mediaconch_command_uuid,
+        tool=new_mediaconch_tool,
+        description='Validate using MediaConch',
```

Comment:
If we take this approach, consider appending `verbose` or `less verbose` to either one, or both of the commands so that it is clear to the end user what they can select from. I can also see some benefit of having verbose and not verbose available - however we still might need more documentation that clearly describes why one would use one over the other.